templates/game-header.php

```
@@ -42,12 +42,12 @@
             Arena: <?php echo htmlspecialchars($Character->Data['arena_floor']); ?> <BR />
         </div>
         <div class="row">
-            Gold: <?php echo round($Character->Data['gold']); ?> <BR />
-            Iron: <?php echo round($Character->Data['iron']); ?> <BR />
+            Gold: <?php echo human_num($Character->Data['gold']); ?> <BR />
```

PHPStan (2.1.33)

Function human_num not found.
Learn more at https://phpstan.org/user-guide/discovering-symbols

(function.notFound)